cdap-operational-stats-core/src/main/java/co/cask/cdap/operations/hdfs/AbstractHDFSStats.java

```diff
@@ -35,14 +30,12 @@
   protected final Configuration conf;
 
   protected AbstractHDFSStats() {
```

Comment:
Seems like we shouldn't have this constructor. Just require the sub-class to pass in one.